frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -165,22 +164,19 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
       variant="medium"
       isOpen={isOpen}
       onClose={() => onBeforeClose(false)}
+      footer={
+        <DashboardModalFooter
+          submitLabel="Deploy"
+          onSubmit={submit}
+          onCancel={() => onBeforeClose(false)}
+          isSubmitDisabled={!canCreate}
+          error={error}
+          alertTitle="Error creating model server"
+        />
+      }
       showClose
-      actions={[
-        <Button key="submit-model" variant="primary" isDisabled={!canCreate} onClick={submit}>
-          Deploy
-        </Button>,
-        <Button key="cancel-model" variant="secondary" onClick={() => onBeforeClose(false)}>
-          Cancel
-        </Button>,
-      ]}
     >
-      <Form
-        onSubmit={(e) => {
-          e.preventDefault();
-          submit();
-        }}
-      >
+      <Form>
```

Comment:
Leaving the default doesn't cause a reload or anything does it? Seems odd to leave the form wrapper when it's not used. I think it provides styles but we override those styles with `Stack`... perhaps look at removing it?

Reply:
Hmm, i think if we remove the Form component, will the `FormGroup` and `FormField` under still be the same style? We need the label styles for it

Reply:
Ah okay, interesting -- I thought the field managed that -- but a form did a gapping -- I could be misremembering.

Reply:
Double checked, the form group inherits many CSS styles defined in the Form component, so we might still want to keep the Form.

Reply:
So we should keep the Form component right?
If so we can integrate it

Reply:
@DaoDaoNoCode

Reply:
I think so.

Reply:
Perfect!